src/engine/osaka.md

```diff
@@ -0,0 +1,109 @@
+# Engine API -- Osaka
+
+Engine API changes introduced in Osaka.
+
+This specification is based on and extends [Engine API - Prague](./prague.md) specification.
```

Comment:
When new versions of methods are introduced there is usually a statement on how to limit the scope of the previous versions. I think this doc should at least add modification to `engine_getPayloadV4` while `engine_getBlobsV2` is a bit more tricky in that regard, see https://github.com/ethereum/execution-apis/blob/main/src/engine/prague.md#update-the-methods-of-previous-forks.

Reply:
Great callout, added `update-the-methods-of-previous-forks` section for `engine_getblobsv1`.

As for getPayloadV4, previous section already specifies the validation should reject anything that's older than Prague, so it covers Osaka.

Reply:
> As for getPayloadV4, previous section already specifies the validation should reject anything that's older than Prague, so it covers Osaka.

`getPayloadV4` is introduced in Prague and it should also be mentioned that it must not be used for Osaka payloads as it is usually done

Reply:
right! got mixed up with v3, updated